README.md

```diff
@@ -16,3 +17,69 @@ To install dependent modules, double click on ```install_dependencies.py```. Alt
 Please note that you might have to change the LABRADHOST environmental variable after installation. For instructions on how to do this, look up on CampbellGroup wiki.
 
 To install the ok module, you need to go the software folder on the Campbell server and install the corresponding package.
+
+# Conventions
+New code should...  
+* Be Python 2 & 3 compatible, using
+   ```python
+   print(str)
+   ````
+   and
+   ```python
+   except Exception as e:
+   ```
+* Follow pep8  
+* Have default config files in the config directory, with local users specifying configuration settings via an external "config" directory at the  same level as this repository.    
```

Comment:
I am in general trying to get away from config files so I would recommend not using them unless absolutely necessary (lots of overhead)